pwa-shell/src/ui/menu/Drawer.js

```diff
@@ -42,20 +44,33 @@ class Drawer extends Component
       this.props.closeDrawer();
     }
   }
+  
+  handleTouchEnd(event) {
+    if (event.velocity >= 0.15 && event.delta[0] < 0) this.props.closeDrawer(); 
+  }
 
   render() {
-    const state = this.props.isOpen ? 'open' : 'closed';
-
+    const { isOpen } = this.props;
+    const state = isOpen ? 'open' : 'closed';
```

Comment:
please don't call it `state`. it can be misleading because it has a very defined meaning in React